tools/review_tracker.py

```
@@ -0,0 +1,389 @@
+        # First, try to get files that have never been reviewed
+        cursor.execute("""
+            SELECT path FROM reviews
+            WHERE last_reviewed IS NULL
+            LIMIT 1
+        """)
+
+        result = cursor.fetchone()
+
+        if result:
+            file_path = result[0]
+            print(f"Next file to review (never reviewed):")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)